changelog/30551.txt

```diff
@@ -0,0 +1,4 @@
+```release-note:bug
```

Comment:
I'm treating this as a bug though it's unclear if we intended for this to work. Let me know if it fits better as an improvement

Reply:
Updated to `change` since it is a difference in behavior that could cause previously passing requests to now fail